### DISCUSSION TITLE
Welcome to hb-appstore Discussions!

### DISCUSSION THREAD

**Discussion by vgmoose** [vgmoose/hb-appstore#96] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by pwsincd** [comment]
Hi my name is pwsincd and am part of the 4TU team and primary maintainer of the appstore along with noahabc12345 , should you have any ideas / issues or feedback regarding the appstore content , i would love to hear your thoughts.
